docker/etna-apache/httpd.conf

```diff
@@ -62,7 +62,7 @@ LoadModule filter_module modules/mod_filter.so
 #LoadModule substitute_module modules/mod_substitute.so
 #LoadModule sed_module modules/mod_sed.so
 #LoadModule charset_lite_module modules/mod_charset_lite.so
-#LoadModule deflate_module modules/mod_deflate.so
+LoadModule deflate_module modules/mod_deflate.so

```

Comment:
why these deflate-related changes?

Reply:
This would only affect dev, should help make things load faster in dev, too.